portfolio/src/main/webapp/script.js

```diff
@@ -46,3 +46,37 @@ function createListElement(text) {
   liElement.innerText = text;
   return liElement;
 }
+
+
+
+function Translate() {
```

Comment:
The function name here is capitalized, which is inconsistent with your other functions. It would be preferable to follow a common convention (e.g. `translate()`).

Reply:
Thank you so much! I fixed it already